### ISSUE TITLE
label powerset broken

### ISSUE THREAD

**Issue by ChristianSch** [scikit-multilearn/scikit-multilearn#30]
```
scikit-multilearn/skmultilearn/meta/lp.py in fit(self, X, y)
     24         last_id = 0
     25         train_vector    = []
---> 26         for labels_applied in y_lil.rows:
     27             label_string = ",".join(map(str,labels_applied))
     28 

NameError: global name 'y_lil' is not defined
```

**Comment by niedakh** [comment]
this is fixed in 71c4f971119d65788a20f2707a3d094580951f29

**Comment by ChristianSch** [comment]
Ok, that's the one I haven't merged yet, sorry.
